@xen-orchestra/backups/.USAGE.md

```diff
@@ -0,0 +1,12 @@
+## Changing the default backup configuration
+
+It is possible to change the default backup configuration by editing the XO configuration file (`config.toml`).
+
+There are multiple sections concerning backups:
+
+- `[backups]` contains values related to the general backup configuration
```

Comment:
This is not related to this lib but to `xo-server` and `xo-proxy`, it is better to add a comment in there respective config saying that this section is passed to this lib and link to this lib's README.